django_tasks/backends/base.py

```diff
@@ -29,6 +32,18 @@ class BaseTaskBackend(metaclass=ABCMeta):
 
     def __init__(self, options: dict) -> None:
         self.alias = options["ALIAS"]
+        self.enqueue_on_commit = options.get("ENQUEUE_ON_COMMIT", None)
```

Comment:
Question: Does it make sense to have a "backend-dependent" option (`None` here)?

Or instead, should we declare that all backends must wait until the end of the transaction by default, and let them deal with implementing a `"ENQUEUE_ON_COMMIT": False` case? 

This is roughly how the `DatabaseBackend` works, as it's not possible to implement the `False` case assuming the same database is used.

Reply:
No.